.gitignore

```diff
@@ -0,0 +1,24 @@
+# Created by .ignore support plugin (hsz.mobi)
+
+### Dev Tools ###
+.idea/*
+
+### macOS ###
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+### Python ###
+# Byte-compiled / optimized / DLL files
+__pycache__/
+
+# Environments
+.env
+.venv
+env/
+venv/
```

Comment:
Thanks for adding the file!
I think we don't need this and other venv things because we aren't really using `venv`.
Pipenv creates the virtual environment files in a separate location locally, so there's no clutter in the repo.